packages/prime/src/utils/files.ts

```diff
@@ -0,0 +1,23 @@
+import heic2any from "heic2any";
+
+export const isHEICFile = (file: File): boolean => {
+    return (
+        file.type.toLowerCase() === "image/heic" ||
+        file.name.toLowerCase().includes(".heic")
+    );
+};
+
+export const convertHeic = async (file: File): Promise<File> => {
+    const blob = await heic2any({
+        blob: file,
+        toType: `image/png`,
+        quality: 1,
+    });
+    return new File(
+        Array.isArray(blob) ? blob : [blob],
+        file.name.replace("heic", "png"),
```

Comment:
this should be only at the end of the file right? what id the name is something like `my-awesome-heic-sample.heic`?